SemanticColorizer/SemanticColorizer.csproj

```diff
@@ -24,6 +25,7 @@
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
     <TargetFrameworkProfile />
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
```

Comment:
I don't think this is required and could interfere with building the solution on existing VS versions.

Reply:
I wasn't sure why VS2019 would put this in myself. But if you remove it and reopen the project in 2019 it "requires an update". I don't remember VS being so aggressive about versions.

Reply:
That's odd. I suppose you can still open/build it in VS 2017 for now?

Reply:
Yep. It opens and builds in VS2017 with no problems. Unfortunately I don't have any older versions to test this with.